consensus/reactor_test.go

```diff
@@ -140,11 +146,21 @@ func TestVotingPowerChange(t *testing.T) {
 }
 
 func TestValidatorSetChanges(t *testing.T) {
+	// FIXME: need to block css[j] until we check all txs! otherwise validator
```

Comment:
@ebuchman will need your vote on this or suggestions. 

note: previously we were blocking the consensus state by blocking the events loop (they were executing in the same thread). new events package runs in its own thread. this is where callback vs channels plays a big role. because you can't exchange msg within the same thread/goroutine.